main.cpp

```diff
@@ -438,27 +543,34 @@ Thing 8) steering wheel
  */
 struct SteeringWheel
 {
-    //wheel diameter
+    
     int wheelDiameter = 60;
-    //wheel circumference
     int wheelCircumference = 10;
-    //rotation amount
     int rotationAmount = 360;
-    //hands free kit controls
     int handsFreeKit = 2;
-    //entertainment controls
     int entertainmentControl = 1;
 
-    //steer truck
+    
     void steerTruck(int degrees);
-    //instrument padel control
-    void instrumentPadelControl();
-    //hands free control 
-    void handsFreeControl();
+    void instrumentPadelControl(bool padelControl);
+    void handsFreeControl(bool remote);
 
 };
 
+void SteeringWheel::steerTruck(int degrees)
+{
+    degrees = 90;
+}
+
+void SteeringWheel::instrumentPadelControl(bool padelControl)
+{
+padelControl =true;
+}
 
+void SteeringWheel::handsFreeControl( bool remote)
+{
+remote = true;
+}
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise